components/support/rc_crypto/nss/Cargo.toml

```diff
@@ -15,7 +15,9 @@ error-support = { path = "../../error" }
 nss_sys = { path = "nss_sys" }
 serde = "1"
 serde_derive = "1"
+once_cell = "1.20.2"
```

Comment:
I think our min versions are such that we can now use `std::cell::OnceCell` and don't actually need this? (But if we did, it looks like it should be an optional dep gated on your new feature)

Reply:
I tried to use  `std::cell::OnceCell` but rustc yelled

```
23 | static NSS_PROFILE_PATH: OnceCell<String> = OnceCell::new();
   |                          ^^^^^^^^^^^^^^^^ `std::cell::OnceCell<std::string::String>` cannot be shared between threads safely
```

Reply:
changed the dependency to be optional, and controlled by the `keydb` feature